genie-web/src/test/resources/application-integration.yml

```diff
@@ -17,13 +17,14 @@
 ##
 
 genie:
+  health:
+    memory:
+      usedPhysicalMemoryPercent: 100
```

Comment:
Instance memory is indeterministic where these integration tests are run. So setting 100 will be safe to have all the integration tests run successfully always.